Makefile

```diff
@@ -7,12 +7,13 @@ top_srcdir		?= ../..
 
 include $(top_srcdir)/include/mk/env_pre.mk
 
-INSTALL_DIR		:= $(prefix)/runltp-ng.d
+BASE_DIR		:= $(abspath $(DESTDIR)/$(prefix))
+INSTALL_DIR		:= $(BASE_DIR)/runltp-ng.d
 
 install:
 	mkdir -p $(INSTALL_DIR)/ltp
```

Comment:
@acerv @richiejp When I manually test the patch, it works.
```
Reviewed-by: Petr Vorel <pvorel@suse.cz>
Tested-by: Petr Vorel <pvorel@suse.cz>
```
Verifying also on OBS: https://build.opensuse.org/project/show/home:pevik:branches:benchmark:ltp:devel